Dockerfile

```diff
@@ -79,10 +79,10 @@ RUN \
       imagemagick \
       enchant \
       clang-format-6.0 \
-      python3.7 python3.7-dev python3.7-distutils \
```

Comment:
Python 3.7 is no longer working, the build keep reporting:

![image](https://github.com/user-attachments/assets/f83ae91b-8209-47b5-965c-6121ffa31c76)

Reply:
Python 3.7 and python 3.8 are now at EOL